materials/knitr.md

```diff
@@ -169,10 +166,10 @@ knitr::kable(results, caption = "Table 1")
 
 <pre><code>Untitled
 ========================================================
-author: 
-date: 
+author: Katherine Johnson
```

Comment:
Why was this the only part of the template you edited? And it seems a little weird that you chose a random name?